manifests/0000_08_cluster-dns-operator_03-cluster-operator.yaml

```diff
@@ -4,6 +4,4 @@ metadata:
   name: dns
 spec: {}
 status:
-  versions:
-    - name: operator
-      version: "0.0.1-snapshot"
\ No newline at end of file
+ # status is set at runtime
```

Comment:
`status: {}` would be more consistent, though I'm not sure whether it's more correct.  However, I'm not sure we want to remove the `versions:` stanz (I asked Derek about it here: https://github.com/openshift/cluster-dns-operator/pull/78#discussion_r261258848).